dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -69,7 +74,12 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
             {
                 THROW_IF_FAILED(PushNotifications_RegisterFullTrustApplication(toastAppId.c_str(), GUID_NULL));
 
-                storedComActivatorString = RegisterComActivatorGuidAndAssets(details);
+
+                // TODO: Add permanent displayname and iconuri.
+                wil::unique_cotaskmem_string processName;
+                THROW_IF_FAILED(GetCurrentProcessPath(processName));
+
+                storedComActivatorString = RegisterComActivatorGuidAndAssets(processName.get(), winrt::Uri{ LR"(C:\Windows\System32\WindowsSecurityIcon.png)" });
```

Comment:
REmove hardcoded PNG please. No hardcoding anywhere in our code base

Reply:
Its likely null that you want to pass through since @danielayala94 is doing the work to retrieve that

Reply:
And when you register the asset, check for null and only store in registry if it is non-null or non-empty.

Reply:
Also @danielayala94  You may want to baseline against this PR

Reply:
Removing the hardcoded properties was on me, I have fixed it as part of the displayName/icon inference work.